lib/workerholic/statistics_storage.rb

```diff
@@ -0,0 +1,16 @@
+module Workerholic
+  class StatsStorage
```

Comment:
We should make the `Storage::RedisWrapper.new` a private class method and have the `StatsStorage` class be used directly, not through an instance of it. It'll avoid creating instances of it all over our code and instead just use the class name and class methods.